api/room/schema_query.py

```diff
@@ -210,3 +217,13 @@ def resolve_bookings_analytics_count(self, info, start_date, end_date):  # noqa:
         analytics = RoomAnalyticsRatios.get_bookings_analytics_count(
             self, query, start_date, end_date)
         return analytics
+
+    def resolve_filter_rooms_by_responses(self, info, lower_bound, upper_bound):
+        query = Room.get_query(info)
+        rooms = query.all()
+        filtered_rooms = []
+        for room in rooms:
```

Comment:
Great work @C3real-kill3r.

What do you think about placing this `count = ResponseModel.query.filter_by(room_id=room.id).count()` expression outside of the loop.
I believe you only need to retrieve the count once and then use it in the loop.

Reply:
Hello @RonKbS  Actually I need it in every loop. The loop gives me a room and then the count gives me the number of responses for that room. Since I am counting for different rooms that is the reason it is within the loop. Hope that makes sense.

Reply:
Kindly use more descriptive variable names. What does count represent?